components/ActionSheet/src/private/MDCActionSheetItemTableViewCell.m

```diff
@@ -187,37 +184,13 @@ - (void)setActionTextColor:(UIColor *)actionTextColor {
       actionTextColor ?: [UIColor.blackColor colorWithAlphaComponent:kLabelAlpha];
 }
 
-- (void)setInkColor:(UIColor *)inkColor {
-  _inkColor = inkColor;
-  // If no ink color then reset to the default ink color
-  self.inkTouchController.defaultInkView.inkColor = inkColor ?: RippleColor();
-}
-
 - (void)setRippleColor:(UIColor *)rippleColor {
-  if (rippleColor != nil && (_rippleColor == rippleColor || [_rippleColor isEqual:rippleColor])) {
-    return;
-  }
   _rippleColor = rippleColor;
```

Comment:
Shouldn't this be the same as the line below in order to be `null_resettable`? So then it would be
`_rippleColor = rippleColor ?: RippleColor;`

Reply:
Ah, good catch, will fix.